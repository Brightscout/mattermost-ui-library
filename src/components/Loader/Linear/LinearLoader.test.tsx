import React from 'react';
import {shallow} from 'enzyme';

import LinearLoader from '.';

describe('Linear Loader', () => {
    it('Should render correctly and matches snapshot', () => {
        const component = shallow(<LinearLoader/>);
        expect(component).toMatchSnapshot();
    });
});
