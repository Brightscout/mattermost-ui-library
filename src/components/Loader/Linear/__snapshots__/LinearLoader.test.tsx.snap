// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Linear Loader Should render correctly 1`] = `ShallowWrapper {}`;
